lib/commands/run.js

```diff
@@ -106,9 +106,16 @@ function run(scriptPath, options) {
             return callback(null, context);
           });
       } else if (context.script.config.payload &&
-                 _.isObject(context.script.config.payload) &&
-                 options.payload) {
-        let csvdata = fs.readFileSync(options.payload, 'utf-8');
+                 (_.isObject(context.script.config.payload) ||
+                 options.payload)) {
+        //use config if path set else use command line
+        let csvdata;        
+        if (context.script.config.payload.path){
+           csvdata = fs.readFileSync(context.script.config.payload.path, 'utf-8');
```

Comment:
The path to payload file is meant to be relative to the location of the test script, whereas this will read it relative to the current working directory of the process.